### ISSUE TITLE
Error response from daemon: manifest for neuropoly/sct:sct-v4.2.1-official not found: manifest unknown

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/sct_docker#45]
email from SCT user:
~~~
Hello,

I’m going to partecipate in the spinal cord toolbox workshop in London next week. I already had the sct version 4.0.0 that I run from docker quickstart terminal cause I use a Windows 10 on my laptop and it was working ( even if with some problems with Qtplatforms).

I tried to download the new version 4.2.1 but it gives me this error:

 

costa@LAPTOP-2RE0GQ7I MINGW64 /c/Program Files/Docker Toolbox

$ docker pull neuropoly/sct:sct-v4.2.1-official

Error response from daemon: manifest for neuropoly/sct:sct-v4.2.1-official not found: manifest unknown: manifest unknown

 

Did I do something wrong? Could you help me to update the new version?

 

Thank you

Best regards
~~~

**Comment by lrouhier** [comment]
I am checking this but i think it is actually a discrepancy between the Readme and the name of the version. I'll answer his mail shortly!

**Comment by jcohenadad** [comment]
> I'll answer his mail shortly!

can you also fix the issue (i.e. update the readme). thx!

**Comment by jcohenadad** [comment]
speaking of which: is it possible to point to "latest" instead of a fixed version? that would prevent such problems in the future...

**Comment by lrouhier** [comment]
Good point. Maybe we can tag the latest version on docker hub a certain way.

**Comment by lrouhier** [comment]
The latest tag is a manual tag so that's not really the solution.
 One option is to create two similar docker images for each release: one called sct-latest, and one called sct-X.X.X then the user can always download sct-latest but he can also download a specific version. It does not take much time to generate two tags from one image. That one extra step for us.

**Comment by jcohenadad** [comment]
> One option is to create two similar docker images for each release: one called sct-latest, and one called sct-X.X.X then the user can always download sct-latest but he can also download a specific version. It does not take much time to generate two tags from one image. That one extra step for us.

Yes, I like that option. It will prevent us from having to update the readme. let's do that!

**Comment by jcohenadad** [comment]
useful doc:
- https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375
- https://blogs.msdn.microsoft.com/stevelasker/2018/03/01/docker-tagging-best-practices-for-tagging-and-versioning-docker-images/
- https://medium.com/@mccode/using-semantic-versioning-for-docker-image-tags-dfde8be06699

@lrouhier i suggest you use simple tags: `latest`, `4.2.1`, `4.0.0`, etc.
